### DISCUSSION TITLE
When using K6, a specific gesture is used to emit a blurred screen

### DISCUSSION THREAD

**Discussion by losehu** [egzumer/uv-k5-firmware-custom#436] [Q&A]
I have found a way to trigger the splash screen 100%:
1. Using K6, K5 does not work
2. Place your palm in front of the machine, firing while approaching the metal shell
3. The official firmware will not cause screen flicker
The video shows my operation, which seems to be related to the metal shell of K6, but I don't understand the specific principle, especially when using official firmware that won't cause screen blur

https://github.com/egzumer/uv-k5-firmware-custom/assets/52628935/a9fe1a78-2852-44fc-96b1-a111cddc74c9

https://github.com/egzumer/uv-k5-firmware-custom/assets/52628935/98b3af38-c3ac-494a-8c65-5de5a3b404de

**Comment by ErikS-web** [comment]
Solved with "screen initialization" after each PTT-release. in [Release 0.16](https://github.com/egzumer/uv-k5-firmware-custom/releases/tag/v0.16) 
But that of course _is an workaround._

Connect the metal cap to Ground (metal chassis inside the radio) is the solution. I tested and that functions

The two bottom ones come through the front.  The upper two are fake.
My idead is to direct the ground via a screw to the plate. 
I just don't have a nice/solid mechanical solution yet.

**Comment by Honkson** [comment]
Hello, set in menu Nr. 32 "MicBar" OFF. If it is ON, i got same Problem! 
I hope i can help you!

**Comment by ErikS-web** [comment]
NO, the **problem is not firmware** but _**HF-interence** by your own radio._

Solution is to GROUND the metal front.

Info on [Wikipedia about radio-frequency interference (RFI)](https://en.wikipedia.org/wiki/Electromagnetic_interference)

**Reply by wc7086** [reply]
It is inexplicable that the official firmware does not have this issue despite the hardware being the same.

**Reply by egzumer** [reply]
The interference is the same but stock firmware doesn't write to LCD while TXing so it doesn't receive corrupted data.

**Comment by wc7086** [comment]
@egzumer

**Comment by prokrypt** [comment]
Try changing the antenna. Stock antenna is not tuned correctly for ham bands and will cause interference issues inside the radio.
